--- Cecil/GodsPVZRuntime1__6.cpp

+++ Raw/GodsPVZRuntime1__6.cpp

@@ -27851,16 +27851,16 @@

 		V_3 = (bool)((((RuntimeObject*)(Board_tF18813446EEAD3B0F908C3ED33FA6B77DC6AAC70*)L_0) == ((RuntimeObject*)(RuntimeObject*)NULL))? 1 : 0);
 		bool L_1 = V_3;
 		if (L_1)
 		{
-			goto IL_008c;
-		}
-	}
-	{
-		goto IL_001b;
-	}
-
-IL_001b:
+			goto IL_0094;
+		}
+	}
+	{
+		goto IL_001f;
+	}
+
+IL_001f:
 	{
 		Board_tF18813446EEAD3B0F908C3ED33FA6B77DC6AAC70* L_2 = __this->___board;
 		int32_t L_3 = ___0_gridX;
 		int32_t L_4 = ___1_gridY;
@@ -27872,16 +27872,16 @@

 		V_6 = (bool)((((RuntimeObject*)(Grid_t2F8C6B2D9E45BAB426C12F3F92E6D48807D08A29*)L_6) == ((RuntimeObject*)(RuntimeObject*)NULL))? 1 : 0);
 		bool L_7 = V_6;
 		if (L_7)
 		{
-			goto IL_0086;
-		}
-	}
-	{
-		goto IL_004b;
-	}
-
-IL_004b:
+			goto IL_008e;
+		}
+	}
+	{
+		goto IL_0053;
+	}
+
+IL_0053:
 	{
 		Grid_t2F8C6B2D9E45BAB426C12F3F92E6D48807D08A29* L_8 = V_4;
 		NullCheck(L_8);
 		bool L_9;
@@ -27891,29 +27891,29 @@

 		V_8 = (bool)((((int32_t)L_10) == ((int32_t)0))? 1 : 0);
 		bool L_11 = V_8;
 		if (L_11)
 		{
-			goto IL_0086;
-		}
-	}
-	{
-		goto IL_0072;
-	}
-
-IL_0072:
+			goto IL_008e;
+		}
+	}
+	{
+		goto IL_007a;
+	}
+
+IL_007a:
 	{
 		int32_t L_12 = ___0_gridX;
 		int32_t L_13 = ___1_gridY;
 		Device_Placing_m744CE374EF05E8B09D6E2003E4EB5063F89FD7EE(__this, L_12, L_13, NULL);
 		return (bool)((int32_t)1);
 	}
 
-IL_0086:
+IL_008e:
 	{
 		return (bool)((int32_t)0);
 	}
 
-IL_008c:
+IL_0094:
 	{
 		NullReferenceException_tBDE63A6D24569B964908408389070C6A9F5005BB* L_14 = (NullReferenceException_tBDE63A6D24569B964908408389070C6A9F5005BB*)il2cpp_codegen_object_new(((RuntimeClass*)il2cpp_codegen_initialize_runtime_metadata_inline((uintptr_t*)&NullReferenceException_tBDE63A6D24569B964908408389070C6A9F5005BB_il2cpp_TypeInfo_var)));
 		NullReferenceException__ctor_mD26D62094A5E49C18D817817E17FDFBC1D3BD752(L_14, NULL);
 		V_5 = L_14;
